applications/matter_bridge/sample.yaml

```diff
@@ -99,7 +99,7 @@ tests:
     extra_args: CONFIG_BRIDGE_GENERIC_SWITCH_BRIDGED_DEVICE=n
       CONFIG_BRIDGE_ONOFF_LIGHT_SWITCH_BRIDGED_DEVICE=y
       CONFIG_CHIP_MEMORY_PROFILING=y CONFIG_BRIDGED_DEVICE_BT=y
-      CONFIG_BRIDGE_MIGRATE_VERSION_1=n
+      CONFIG_BRIDGE_MIGRATE_VERSION_1=n CONFIG_WIFI_NM_WPA_SUPPLICANT_WPA3=n
```

Comment:
Do you know what is the memory reduction due to disabling WPA3?

Reply:
15.6KB of FLASH, 320 bytes of RAM.